cmd/finalize/finalize.go

```diff
@@ -155,9 +156,13 @@ func main() {
 					message.AccessionID,
 					message.DecryptedChecksums,
 					err)
-
+				requeueMsg := true
+				if !strings.Contains(err.Error(), "duplicate key value violates unique constraint") {
+					requeueMsg = false
+				}
```

Comment:
Ask not to requeue if the error isn't duplicate key - keep requeueing if it is?

Reply:
yes

Reply:
:facepalm: did not see the `!`